presto-clp/src/main/java/com/facebook/presto/plugin/clp/ClpExpression.java

```diff
@@ -27,13 +27,15 @@ public class ClpExpression
 {
     // Optional KQL query or column name representing the fully or partially translatable part of the expression.
     private final Optional<String> pushDownExpression;
-
+    // Optional SQL string extracted from the query plan, which is only made of given metadata columns.
```

Comment:
```suggestion

    // Optional SQL string extracted from the query plan, which is only made of given metadata columns.
```

Remember that we keep an empty line before each comment block.